gravitee-apim-console-webui/src/components/gio-top-nav/gio-top-nav.component.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Inject, OnDestroy, OnInit } from '@angular/core';
+import { StateService } from '@uirouter/core';
+import { IRootScopeService } from 'angular';
+import { TransitionService } from '@uirouter/angularjs';
+import { Subject, timer } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+import { Constants } from '../../entities/Constants';
+import { AjsRootScope, CurrentUserService, UIRouterState } from '../../ajs-upgraded-providers';
+import UserService from '../../services/user.service';
+import { User } from '../../entities/user/user';
+import { TaskService } from '../../services-ngx/task.service';
+
+@Component({
+  selector: 'gio-top-nav',
+  template: require('./gio-top-nav.component.html'),
+  styles: [require('./gio-top-nav.component.scss')],
+})
+export class GioTopNavComponent implements OnInit, OnDestroy {
+  private unsubscribe$ = new Subject();
+  public displayDocumentationButton = false;
+  public hasAlert = false;
+  public currentUser: User;
+  public userTaskCount = 0;
+  public userShortName: string;
+  public supportEnabled: boolean;
+  public newsletterProposed: boolean;
+
+  constructor(
+    @Inject(UIRouterState) private readonly ajsState: StateService,
+    @Inject(AjsRootScope) private readonly ajsRootScope: IRootScopeService,
+    @Inject('Constants') public readonly constants: Constants,
+    @Inject(CurrentUserService) private readonly currentUserService: UserService,
+    public readonly taskService: TaskService,
+    public readonly $transitions: TransitionService,
+  ) {}
+
+  ngOnInit(): void {
+    this.currentUser = this.currentUserService.currentUser;
+    this.newsletterProposed =
+      (this.currentUser && !this.currentUser.firstLogin) ||
+      !!window.localStorage.getItem('newsletterProposed') ||
+      !this.constants.org.settings.newsletter.enabled;
+    this.supportEnabled = this.constants.org.settings.management.support.enabled;
+
+    timer(0, this.taskService.getTaskSchedulerInSeconds() * 1000)
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe(() => {
+        this.taskService.getTasks().subscribe((taskPagedResult) => {
+          this.userTaskCount = taskPagedResult.page.total_elements;
+          this.hasAlert = this.userTaskCount > 0;
+        });
+      });
+
+    this.$transitions.onFinish({}, (trans) => {
```

Comment:
i'm wonder if we can't use like in api-navigation : 
```
this.ajsState.includes(baseRoute)
```
to avoid angularJs $transitions use 🤔 and have simple array liste to some